core/const.cpp

```diff
@@ -4,9 +4,97 @@
  */
 
 #include "const.hpp"
-
-#define DEFINE(x) decltype(x) x
+#include "common/error_text.hpp"
+#include "primitives/sector/sector.hpp"
+#include "vm/actor/builtin/types/market/policy.hpp"
+#include "vm/actor/builtin/types/miner/policy.hpp"
+#include "vm/actor/builtin/types/payment_channel/policy.hpp"
+#include "vm/actor/builtin/types/storage_power/policy.hpp"
+#include "vm/actor/builtin/types/verified_registry/policy.hpp"
 
 namespace fc {
-  DEFINE(kConsensusMinerMinPower){BigInt{1} << 40};
+  using primitives::sector::RegisteredSealProof;
+
+  /**
+   * Class for profile name validation with boost::program_options
+   */
+  class Profile : public std::string {};
```

Comment:
Why not `using Profile = std::string`?

Reply:
fixed

Reply:
Actually `using Profile = std::string` creates type alias, but not the new type required for validation.